server/syncSubs.ts

```diff
@@ -53,21 +56,35 @@ async function syncSubscribers() {
     status: sub.status,
     uid: uidMap.get(emailMap.get(sub.customer)),
   }));
+  currentSubs = result
+    .map((sub) => sub.uid)
+    .sort()
+    .join();
 
   // Upsert to DB
   // console.log(result);
-  try {
-    await postgres2?.query('BEGIN TRANSACTION');
-    await postgres2?.query('DELETE FROM subscriber');
-    for (let i = 0; i < result.length; i++) {
-      const row = result[i];
-      await insertObject(postgres2, 'subscriber', row);
+  if (currentSubs !== lastSubs) {
+    try {
+      await postgres2?.query('BEGIN TRANSACTION');
+      await postgres2?.query('DELETE FROM subscriber');
+      await postgres2?.query('UPDATE room SET "isSubRoom" = false');
+      for (let i = 0; i < result.length; i++) {
+        const row = result[i];
+        await insertObject(postgres2, 'subscriber', row);
+        await updateObject(
+          postgres2,
+          'room',
+          { isSubRoom: true },
+          { owner: row.uid }
+        );
+      }
+      await postgres2?.query('COMMIT');
+      console.log('%s subscribers', result.length);
```

Comment:
Can we move this out of the if block so it occasionally prints the number of subs? I sometimes debug the process by seeing if this is running consistently.